### PR TITLE
docs: Prepare Changelog for 7.10.0

### PR DESCRIPTION
**Pull request by andresrc**
Prepare Changelog for 7.10.0.

Merge after first BC.

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -3,6 +3,283 @@
...
+=== Beats version 7.10.0
+https://github.com/elastic/beats/compare/v7.9.3...v7.10.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Added `certificate` TLS verification mode to ignore server name mismatch. {issue}12283[12283] {pull}20293[20293]
+- Remove redundant `cloudfoundry.*.timestamp` fields. This value is set in `@timestamp`. {pull}21175[21175]
+- Allow embedding of CAs, Certificate of private keys for anything that support TLS in ouputs and inputs https://github.com/elastic/beats/pull/21179
```
Comment (v1v) [inline CHANGELOG.asciidoc:16]:
```suggestion
- Allow embedding of CAs, Certificate of private keys for anything that support TLS in outputs and inputs {pull}21179[21179]
```

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -3,6 +3,283 @@
...
+- Remove io.time from windows {pull}22237[22237]
+
+*Packetbeat*
+
+- Add "network" to event.category {issue}20364[20364] {pull}20392[20392]
+
+*Winlogbeat*
+
+- Fix invalid IP addresses in DNS query results from Sysmon data. {issue}18432[18432] {pull}18436[18436]
+- Fix `event.outcome` in the security module for non-English languages. {issue}20079[20079] {pull}20564[2056
```
Comment (v1v) [inline CHANGELOG.asciidoc:133]:
```suggestion
- Fix `event.outcome` in the security module for non-English languages. {issue}20079[20079] {pull}20564[2056]
```

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -3,6 +3,283 @@
...
+- Fields from Winlogbeat modules were not being included in index templates and patterns. {pull}18983[18983]
+- Protect against accessing undefined variables in Sysmon module. {issue}22219[22219] {pull}22236[22236]
+
+*Functionbeat*
+
+- Fix catchall bucket config errors by adding more validation. {issue}17572[16282] {pull}20887[16287]
```
Comment (v1v) [inline CHANGELOG.asciidoc:139]:
```suggestion
- Fix catchall bucket config errors by adding more validation. {issue}17572[17572] {pull}20887[20887]
```

File: libbeat/docs/release.asciidoc [hunk 4]
```diff
@@ -8,6 +8,7 @@ This section summarizes the changes in each release. Also read
 <<breaking-changes>> for more detail about changes that affect
 upgrade.
 
+* <<release-notes-7.10.0>>
```
Comment (v1v) [inline libbeat/docs/release.asciidoc:11]:
```suggestion
* <<release-notes-7.10.0>>
* <<release-notes-7.9.3>>
```

**Review by v1v — COMMENTED** [review]
Some typos

**Comment by andresrc** [comment]
Thanks @v1v !!

File: CHANGELOG.asciidoc [hunk 5]
```diff
@@ -3,6 +3,283 @@
...
+
+*Heartbeat*
+
+
+*Journalbeat*
+
+
+*Metricbeat*
+
+- Add support for azure light metricset app_stats. {pull}20639[20639]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:108]:
```suggestion
- Add support for azure light metricset `app_stats`. {pull}20639[20639]
```

File: CHANGELOG.asciidoc [hunk 6]
```diff
@@ -3,6 +3,283 @@
...
+- Fix retrieving resources by ID for the azure module. {pull}21711[21711] {issue}21707[21707]
+- Use timestamp from CloudWatch API when creating events. {pull}21498[21498]
+- Report the correct windows events for system/filesystem {pull}21758[21758]
+- Fix regular expression in windows/permfon. {pull}22146[22146] {issue}21125[21125]
+- Fix azure storage event format. {pull}21845[21845]
+- Fix panic in kubernetes autodiscover related to keystores {issue}21843[21843] {pull}21880[21880]
+- [Kubernetes] Remove redundant dockersock volume mount {pull}22009[22009]
+- Revert change to report `process.memory.rss` as `process.memory.wss` on Windows. {pull}22055[22055]
+- Add interval information to `monitor` metricset in azure. {pull}22152[22152]
+- Remove io.time from windows {pull}22237[22237]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:124]:
```suggestion
- Remove `io.time` from windows {pull}22237[22237]
```

File: CHANGELOG.asciidoc [hunk 7]
```diff
@@ -3,6 +3,283 @@
...
+
+*Packetbeat*
+
+- Add "network" to event.category {issue}20364[20364] {pull}20392[20392]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:128]:
```suggestion
- Add "network" to `event.category` {issue}20364[20364] {pull}20392[20392]
```

File: CHANGELOG.asciidoc [hunk 8]
```diff
@@ -3,6 +3,283 @@
...
+
+==== Added
+
+*Affecting all Beats*
+
+- Add minimum cache TTL for successful DNS responses. {pull}18986[18986]
+- Add support for DNS over TLS for the `dns` processor. {pull}19321[19321]
+- Add leader election for Kubernetes autodiscover. {pull}20281[20281]
+- Add capability of enriching process metadata with contianer id also for non-privileged containers in `add_process_metadata` processor. {pull}19767[19767]
+- Add replace_fields config option in add_host_metadata for replacing host fields. {pull}20490[20490] {issue}20464[20464]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:150]:
```suggestion
- Add `replace_fields` config option in `add_host_metadata` for replacing host fields. {pull}20490[20490] {issue}20464[20464]
```

File: CHANGELOG.asciidoc [hunk 9]
```diff
@@ -3,6 +3,283 @@
...
+- Added experimental dataset `barracuda/spamfirewall`. {pull}20820[20820]
+- Added experimental dataset `cisco/meraki`. {pull}20820[20820]
+- Added experimental dataset `f5/bigipafm`. {pull}20820[20820]
+- Added experimental dataset `fortinet/fortimail`. {pull}20820[20820]
+- Added experimental dataset `fortinet/fortimanager`. {pull}20820[20820]
+- Added experimental dataset `juniper/netscreen`. {pull}20820[20820]
+- Added experimental dataset `sophos/utm`. {pull}20820[20820]
+- Add Cloud Foundry tags in related events. {pull}21177[21177]
+- Cloud Foundry metadata is cached to disk. {pull}20775[20775]
+- Add option to select the type of index template to load: legacy, component, index. {pull}21212[21212]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:166]:
```suggestion
- Add option to select the type of index template to load: `legacy`, `component`, `index`. {pull}21212[21212]
```

File: CHANGELOG.asciidoc [hunk 10]
```diff
@@ -3,6 +3,283 @@
...
+
+*Auditbeat*
+
+- Add enrichment of auditd seccomp events with name of the architecture, syscall, and signal. {issue}14055[14055] {pull}19300[19300]
+
+*Filebeat*
+
+
+- Add support for reading auditd logs that are prefixed with `node=`. {pull}19659[19659]
+- Add event.ingested to all Filebeat modules. {pull}20386[20386]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:177]:
```suggestion
- Add `event.ingested` to all Filebeat modules. {pull}20386[20386]
```

File: CHANGELOG.asciidoc [hunk 11]
```diff
@@ -3,6 +3,283 @@
...
+- Add event.ingested for Suricata module {pull}20220[20220]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:178]:
```suggestion
- Add `event.ingested` for Suricata module {pull}20220[20220]
```

File: CHANGELOG.asciidoc [hunk 12]
```diff
@@ -3,6 +3,283 @@
...
+- Add support for custom header and headersecret for filebeat http_endpoint input {pull}20435[20435]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:179]:
```suggestion
- Add support for custom header and headersecret for filebeat `http_endpoint` input {pull}20435[20435]
```

File: CHANGELOG.asciidoc [hunk 13]
```diff
@@ -3,6 +3,283 @@
...
+- Convert httpjson to v2 input {pull}20226[20226]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:180]:
```suggestion
- Convert `httpjson` to v2 input {pull}20226[20226]
```

File: CHANGELOG.asciidoc [hunk 14]
```diff
@@ -3,6 +3,283 @@
...
+- Add event.ingested to all Filebeat modules. {pull}20386[20386]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:181]:
```suggestion
- Add `event.ingested` to all Filebeat modules. {pull}20386[20386]
```

File: CHANGELOG.asciidoc [hunk 15]
```diff
@@ -3,6 +3,283 @@
...
+- Return error when log harvester tries to open a named pipe. {issue}18682[18682] {pull}20450[20450]
+- Avoid goroutine leaks in Filebeat readers. {issue}19193[19193] {pull}20455[20455]
+- Improve Zeek x509 module with `x509` ECS mappings {pull}20867[20867]
+- Improve Zeek SSL module with `x509` ECS mappings {pull}20927[20927]
+- Added new properties field support for event.outcome in azure module {pull}20998[20998]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:186]:
```suggestion
- Added new properties field support for `event.outcome` in azure module {pull}20998[20998]
```

File: CHANGELOG.asciidoc [hunk 16]
```diff
@@ -3,6 +3,283 @@
...
+- Improve Zeek Kerberos module with `x509` ECS mappings {pull}20958[20958]
+- Improve Fortinet firewall module with `x509` ECS mappings {pull}20983[20983]
+- Improve Santa module with `x509` ECS mappings {pull}20976[20976]
+- Improve Suricata Eve module with `x509` ECS mappings {pull}20973[20973]
+- Added new module for Zoom webhooks {pull}20414[20414]
+- Add type and sub_type to panw panos fileset {pull}20912[20912]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:192]:
```suggestion
- Add `type` and `sub_type` to panw `panos` fileset {pull}20912[20912]
```

File: CHANGELOG.asciidoc [hunk 17]
```diff
@@ -3,6 +3,283 @@
...
+- Always attempt community_id processor on zeek module {pull}21155[21155]
+- Add related.hosts ecs field to all modules {pull}21160[21160]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:194]:
```suggestion
- Add `related.hosts` ecs field to all modules {pull}21160[21160]
```

File: CHANGELOG.asciidoc [hunk 18]
```diff
@@ -3,6 +3,283 @@
...
+- Keep cursor state between httpjson input restarts {pull}20751[20751]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:195]:
```suggestion
- Keep cursor state between `httpjson` input restarts {pull}20751[20751]
```

File: CHANGELOG.asciidoc [hunk 19]
```diff
@@ -3,6 +3,283 @@
...
+- Convert aws s3 to v2 input {pull}20005[20005]
+- Add support for additional fields from V2 ALB logs. {pull}21540[21540]
+- Release Cloud Foundry input as GA. {pull}21525[21525]
+- New Cisco Umbrella dataset {pull}21504[21504]
+- New juniper.srx dataset for Juniper SRX logs. {pull}20017[20017]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:200]:
```suggestion
- New `juniper.srx` dataset for Juniper SRX logs. {pull}20017[20017]
```

File: CHANGELOG.asciidoc [hunk 20]
```diff
@@ -3,6 +3,283 @@
...
+*Heartbeat*
+
+*Journalbeat*
+
+
+*Metricbeat*
+
+- Add `state_statefulset` metricset to Metricbeat recommended configuration for k8s. {pull}17627[17627]
+- Infer types in Prometheus remote_write. {pull}19944[19944]
+- Add cloud.instance.name into aws ec2 metricset. {pull}20077[20077]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:218]:
```suggestion
- Add `cloud.instance.name` into aws ec2 metricset. {pull}20077[20077]
```

File: CHANGELOG.asciidoc [hunk 21]
```diff
@@ -3,6 +3,283 @@
...
+- Add host inventory metrics into aws ec2 metricset. {pull}20171[20171]
+- Add `scope` setting for elasticsearch module, allowing it to monitor an Elasticsearch cluster behind a load-balancing proxy. {issue}18539[18539] {pull}18547[18547]
+- Add state_daemonset metricset for Kubernetes Metricbeat module {pull}20649[20649]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:221]:
```suggestion
- Add `state_daemonset` metricset for Kubernetes Metricbeat module {pull}20649[20649]
```

File: CHANGELOG.asciidoc [hunk 22]
```diff
@@ -3,6 +3,283 @@
...
+- Add host inventory metrics to googlecloud compute metricset. {pull}20391[20391]
+- Add host inventory metrics to azure compute_vm metricset. {pull}20641[20641]
+- Add host inventory metrics to system module. {pull}20415[20415]
+- Add billing data collection from Cost Explorer into aws billing metricset. {pull}20527[20527] {issue}20103[20103]
+- Migrate `compute_vm` metricset to a light one, map `cloud.instance.id` field. {pull}20889[20889]
+- Request prometheus endpoints to be gzipped by default {pull}20766[20766]
+- Add latency config parameter into aws module. {pull}20875[20875]
+- Add billing metricset into googlecloud module. {pull}20812[20812] {issue}20738[20738]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:229]:
```suggestion
- Add `billing` metricset into googlecloud module. {pull}20812[20812] {issue}20738[20738]
```

File: CHANGELOG.asciidoc [hunk 23]
```diff
@@ -3,6 +3,283 @@
...
+- Release all kubernetes `state` metricsets as GA {pull}20901[20901]
+- Move `compute_vm_scaleset` to light metricset. {pull}21038[21038] {issue}20985[20985]
+- Sanitize `event.host`. {pull}21022[21022]
+- Add support for different Azure Cloud environments in the metricbeat azure module. {pull}21044[21044] {issue}20988[20988]
+- Add overview and platform health dashboards to Cloud Foundry module. {pull}21124[21124]
+- Release lambda metricset in aws module as GA. {issue}21251[21251] {pull}21255[21255]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:235]:
```suggestion
- Release `lambda` metricset in aws module as GA. {issue}21251[21251] {pull}21255[21255]
```

File: CHANGELOG.asciidoc [hunk 24]
```diff
@@ -3,6 +3,283 @@
...
+- Add dashboard for pubsub metricset in googlecloud module. {pull}21326[21326] {issue}17137[17137]
```
Comment (hendry-lim) [inline CHANGELOG.asciidoc:236]:
```suggestion
- Add dashboard for `pubsub` metricset in googlecloud module. {pull}21326[21326] {issue}17137[17137]
```

**Review by hendry-lim — CHANGES_REQUESTED** [review]
Added missing code formatting

File: CHANGELOG.asciidoc [hunk 25]
```diff
@@ -3,6 +3,283 @@
...
+=== Beats version 7.10.0
+https://github.com/elastic/beats/compare/v7.9.3...v7.10.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Added `certificate` TLS verification mode to ignore server name mismatch. {issue}12283[12283] {pull}20293[20293]
+- Remove redundant `cloudfoundry.*.timestamp` fields. This value is set in `@timestamp`. {pull}21175[21175]
+- Allow embedding of CAs, Certificate of private keys for anything that support TLS in outputs and inputs {pull}21179[21179]
```
Comment (bmorelli25) [inline CHANGELOG.asciidoc:16]:
```suggestion
- Allow embedding of CAs, Certificate of private keys for anything that supports TLS in outputs and inputs {pull}21179[21179]
```

File: CHANGELOG.asciidoc [hunk 26]
```diff
@@ -3,6 +3,283 @@
...
+*Heartbeat*
+
+
+*Journalbeat*
+
+
+
+*Metricbeat*
+
+- Remove "invalid zero" metrics on Windows and Darwin, don't report linux-only memory and diskio metrics when running under agent. {pull}21457[21457]
```
Comment (bmorelli25) [inline CHANGELOG.asciidoc:51]:
```suggestion
- Remove "invalid zero" metrics on Windows and Darwin, don't report linux-only memory and disk I/O metrics when running under agent. {pull}21457[21457]
```

File: CHANGELOG.asciidoc [hunk 27]
```diff
@@ -3,6 +3,283 @@
...
+- Fix Google Cloud Function configuration issue. {issue}20864[20864] {pull}22156[22156]
+
+==== Added
+
+*Affecting all Beats*
+
+- Add minimum cache TTL for successful DNS responses. {pull}18986[18986]
+- Add support for DNS over TLS for the `dns` processor. {pull}19321[19321]
+- Add leader election for Kubernetes autodiscover. {pull}20281[20281]
+- Add capability of enriching process metadata with contianer id also for non-privileged containers in `add_process_metadata` processor. {pull}19767[19767]
```
Comment (bmorelli25) [inline CHANGELOG.asciidoc:149]:
```suggestion
- Add capability of enriching process metadata with container id also for non-privileged containers in `add_process_metadata` processor. {pull}19767[19767]
```

File: CHANGELOG.asciidoc [hunk 28]
```diff
@@ -3,6 +3,283 @@
...
+*Journalbeat*
+
+
+*Metricbeat*
+
+- Add `state_statefulset` metricset to Metricbeat recommended configuration for k8s. {pull}17627[17627]
+- Infer types in Prometheus remote_write. {pull}19944[19944]
+- Add cloud.instance.name into aws ec2 metricset. {pull}20077[20077]
+- Add host inventory metrics into aws ec2 metricset. {pull}20171[20171]
+- Add `scope` setting for elasticsearch module, allowing it to monitor an Elasticsearch cluster behind a load-balancing proxy. {issue}18539[18539] {pull}18547[18547]
```
Comment (bmorelli25) [inline CHANGELOG.asciidoc:220]:
```suggestion
- Add `scope` setting for Elasticsearch module, allowing it to monitor an Elasticsearch cluster behind a load-balancing proxy. {issue}18539[18539] {pull}18547[18547]
```

File: CHANGELOG.asciidoc [hunk 29]
```diff
@@ -3,6 +3,283 @@
...
+- Fix syslog RFC 5424 parsing in the CheckPoint module. {pull}21854[21854]
+- Fix incorrect connection state mapping in zeek connection pipeline. {pull}22151[22151] {issue}22149[22149]
+
+*Heartbeat*
+
+- Add support for new `service_name` option to all monitors. {pull}19932[19932].
+
+*Heartbeat*
+
+
```
Comment (bmorelli25) [inline CHANGELOG.asciidoc:102]: Duplicate
```suggestion
```

File: CHANGELOG.asciidoc [hunk 30]
```diff
@@ -3,6 +3,283 @@
...
+- Adding support for Microsoft 365 Defender (Microsoft Threat Protection) {pull}21446[21446]
+- Adding support for FIPS in s3 input {pull}21446[21446]
+- Update Okta documentation for new stateful restarts. {pull}22091[22091]
+
+*Heartbeat*
+
+- Add index and pipeline settings to monitor configurations. {pull}20610[20610]
+
+*Heartbeat*
+
```
Comment (bmorelli25) [inline CHANGELOG.asciidoc:210]: Duplicate
```suggestion
```

File: CHANGELOG.asciidoc [hunk 31]
```diff
@@ -3,6 +3,283 @@
...
+*Winlogbeat*
+
+*Functionbeat*
+
+==== Known Issue
+
+*Journalbeat*
+
+
```
Comment (bmorelli25) [inline CHANGELOG.asciidoc:281]: I would get rid of all of these and just add n/a
```suggestion

- N/A
```

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**